.github/workflows/ci.yaml

```diff
@@ -12,19 +12,26 @@ on:
   workflow_dispatch:
 
 env:
-  ormolu_version: 0.5.2.0
-  ucm_local_bin: ucm-local-bin
+  ## Some version numbers that are used during CI
```

Comment:
I feel like some editor told me having comments in here was illegal, but I guess it's not true